partiql-lang/src/test/kotlin/org/partiql/lang/eval/impl/FunctionManagerTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.partiql.lang.eval.impl
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.partiql.lang.eval.ArityMismatchException
+import org.partiql.lang.eval.EvaluationException
+import org.partiql.lang.eval.EvaluatorTestBase
+import org.partiql.lang.eval.FunctionNotFoundException
+import org.partiql.lang.eval.builtins.SCALAR_BUILTINS_DEFAULT
+import org.partiql.types.StaticType
+
+class FunctionManagerTest : EvaluatorTestBase() {
+
+    val functions = SCALAR_BUILTINS_DEFAULT
+    private val functionManager = FunctionManager(functions)
+
+    @Test
+    fun functionManagerPassTests() {
```

Comment:
Great work on these tests.

1. Can you split each of these passing tests into separate tests?
2. Also, can you have test cases for passing in `null`?
3. Can you also have a test corresponding to a subtype? So, since `TRIM`'s first parameter is TEXT, can we attempt to resolve `TRIM(<string>)` ?